.github/workflows/tests.yml

```diff
@@ -10,13 +10,14 @@ on:
     branches:
       - main
       - release-*
-    paths-ignore:
+    paths:
       # This is spelt like this to do not ignore the docs/examples/ folder.
-      - docs/*
-      - docs/usage/*
-      - CHANGELOG.rst
-      - LICENSE
-      - README.md
+      - '**'
+      - '!docs/examples/**'
```

Comment:
This didn't quite go right I think. It now ignores the examples and triggers on other files in `docs/`.

Reply:
Indeed. Fixed, I think.